codalab/rest/interpret.py

```diff
@@ -480,6 +497,8 @@ def interpret_genpath_table_contents(contents):
             # value can be either a string (already rendered) or a (bundle_uuid, genpath, post) triple
             if is_bundle_genpath_triple(value):
                 requests.append(value)
+    if dry_run:
+        return requests
```

Comment:
Hm, the fact that this function returns two types of things is non-ideal...

Reply:
We could make a separate function called `contains_bundle_genpath_triple`, but then we'd have to duplicate some logic -- do you think it's worth it to do that to avoid the problem you mentioned?

Reply:
Maybe extract the requests generation out to a separate function, and call it in this function (replacing line 493-499), and call it directly where the dry_run is supposed to be true?